packages/pros/src/lib.rs

```diff
@@ -377,12 +377,13 @@ pub mod prelude {
         color::Rgb,
         devices::{
             adi::{
-                analog::{AdiAnalogIn, AdiAnalogOut},
```

Comment:
Why was `AdiAnalogOut` removed from the prelude?

Reply:
Leftover from a previous change.